backend/src/main/java/hu/flowacademy/worksheet/service/WorksheetService.java

```diff
@@ -74,4 +74,66 @@ public Worksheet setStatusWorksheet(String id, WorksheetStatus status) throws Va
         toChange.setWorksheetStatus(status);
         return worksheetRepository.save(toChange);
     }
+
+    public Worksheet update(String id, Worksheet worksheet) throws ValidationException {
+        validateUpdatedWorksheet(worksheet);
+        Worksheet updatedWorksheet = worksheetRepository.findById(id).orElseThrow(() -> new ValidationException("The id is null or not real: " + worksheet.getId()));
```

Comment:
Inkább úgy fogalmazd meg a hibát, ahogyan a requestet küldő szempontjából érthető lesz: "No worksheet with the given id xyz"